### PR TITLE
Add proposed Annual Review process discussion

### PR DESCRIPTION
**Pull request by dthaler**
Discussed in April 16 TAC meeting and in issue #18

Signed-off-by: Dave Thaler <dthaler@ntdev.microsoft.com>

### CODE REVIEW

File: project-progression-policy.md [hunk 1]
```diff
@@ -145,5 +145,29 @@ Projects may be granted Emeritus status via a vote of two-thirds of the represen
 
 ## IV. Annual Review Process
 
-The TAC shall develop an annual review process to determine whether projects are in the stage that accurately reflects their needs and goals. 
+The TAC will conduct a review of each accepted project on an annual basis.
 
+The review includes the following:
+
+1. Review whether any answers to the project's submission template have
```
Comment (MikeCamel) [inline project-progression-policy.md:152]: Not all answers are relevant to a running project.  Once the template is complete, we should consider which ones we think should be reviewed.

File: project-progression-policy.md [hunk 2]
```diff
@@ -145,5 +145,29 @@ Projects may be granted Emeritus status via a vote of two-thirds of the represen
...
+   changed, and if so, review the new answers. A representative from the
+   project is responsible for presenting any deltas to the template answers
+   since the last review, if any. If there are no changes, there is nothing
+   to review here.
+
+2. Review the project's progression status to determine whether the project
+   is in the stage that accurately reflects its needs and goals. For example,
+   is it already ready to move to another progression level? Is it on track
+   at the current level? Is any action needed from the TAC (e.g., change in
+   or addition to any project mentor(s))?  If nothing has changed
```
Comment (MikeCamel) [inline project-progression-policy.md:162]: This works both ways - the mentor may request a change, I guess.  Not sure if this is worth calling out.

File: project-progression-policy.md [hunk 3]
```diff
@@ -145,5 +145,29 @@ Projects may be granted Emeritus status via a vote of two-thirds of the represen
...
+   significantly, there may be nothing to review here.
+
+3. Review any budget allocations relevant to the project, and whether any
+   adjustments are needed.
+
+Projects are encouraged to proactively inform the TAC when something
+changes that affects their submission template (changing a License, security
+reporting process, CoC, etc.), rather than waiting for the next annual review.
+
+It is the responsibility of the project's mentor to help the project determine
```
Comment (MikeCamel) [inline project-progression-policy.md:172]: Do the three (nearly four) existing projects all have designated mentors at this point?

File: project-progression-policy.md (hunk 1 above)
Comment (dthaler) [reply project-progression-policy.md:152]: @MikeCamel Sure, but if its not relevant after creation, then the answer won't have "changed" per se so I don't think the text here is incorrect, do you?   Just a request for a future action item I think once we have more clarity.

File: project-progression-policy.md (hunk 2 above)
Comment (dthaler) [reply project-progression-policy.md:162]: @MikeCamel Yes for the three (me, Simon, and you).  Graphene: not yet until it completes the process.

**Comment by dthaler** [comment]
@MikeCamel any objection to merging as is and treating any additions in a future PR?

**Comment by MikeCamel** [comment]
> @MikeCamel any objection to merging as is and treating any additions in a future PR?

None.  Approved.

**Pull request merged**